engine/gamesys/src/gamesys/components/comp_collection_factory.cpp

```diff
@@ -197,9 +229,9 @@ namespace dmGameSystem
         return true;
     }
 
-    bool CompCollectionFactoryUnload(dmGameObject::HCollection collection, CollectionFactoryComponent* component)
+    bool CompCollectionFactoryUnload(CollectionFactoryWorld* world, CollectionFactoryComponent* component)
     {
-        if(!component->m_Resource->m_LoadDynamically)
+        if(!GetResource(component)->m_LoadDynamically)
```

Comment:
Note:
Even if the original factory has "load dynamically = false", we will set it to "true" for the custom prorotype, since it wasn't loaded by the initial load of the collection, but in fact requires you to load the resources.